internal/lookup/lookup.go

```diff
@@ -188,13 +190,13 @@ func (p *Global) GetSymbolOfObject(obj types.Object) (*scip.SymbolInformation, b
 	// 	))
 	// }
 
-	return nil, false, errors.New(fmt.Sprintf(
+	return nil, false, fmt.Errorf(
```

Comment:
nit: Why not use `errors.Newf(`? That would make it clearer that an error is being constructed?

Reply:
errors.Newf isnt from stdlib, do we already have a dependency on github.com/sourcegraph/sourcegraph/lib in scip-go?

Reply:
There is an indirect dependency in `go.mod`, so it seems fine to use that here:

```
	github.com/sourcegraph/sourcegraph/lib v0.0.0-20220511160847-5a43d3ea24eb // indirect
```